packages/report-server/src/__tests__/reportBuilder/output/output.test.ts

```diff
@@ -213,6 +236,89 @@ describe('output', () => {
       });
     });
 
-    // TODO: it('can perform with subCategoryField', () =>{})
+    describe('handle rowField', () => {
+      it('throws error if rowField matches any columns', () => {
+        const output = buildOutput({
+          type: 'matrix',
+          categoryField: 'InfrastructureType',
+          rowField: 'FacilityType',
+          columns: ['FacilityType', 'Laos', 'Tonga'],
+        });
+        expect(() => {
+          output(MULTIPLE_TRANSFORMED_DATA_WITH_CATEGORIES);
+        }).toThrow();
+      });
+
+      it('throws error if rowField matches categoryField', () => {
+        const output = buildOutput({
+          type: 'matrix',
+          categoryField: 'FacilityType',
+          rowField: 'FacilityType',
+        });
+        expect(() => {
+          output(MULTIPLE_TRANSFORMED_DATA_WITH_CATEGORIES);
+        }).toThrow();
+      });
+
+      it('throws error if rowField not provided ', () => {
+        const output = buildOutput({
+          type: 'matrix',
+          categoryField: 'FacilityType',
+        });
+        expect(() => {
+          output(MULTIPLE_TRANSFORMED_DATA_WITH_CATEGORIES);
```

Comment:
Maybe it's just me + Monday morning, but it looks like the function argument MULTIPLE_TRANSFORMED_DATA_WITH_CATEGORIES is never used in any of these calls added in the PR?

Reply:
No, you're totally right. We need to pass in some kind of array of values to this function, but given that we're just testing config validation here it could be an empty array for all intents and purposes. I'll change that 👍